MAINTAINERS.yml

```
@@ -2809,6 +2809,36 @@ Toolchain Integration:
   labels:
     - "area: Toolchains"
 
+Toolchain ARC MWDT:
+  status: maintained
+  maintainers:
+    - evgeniy-paltsev
+    - abrodkin
+  files:
+    - cmake/*/arcmwdt/
```

I guess it worth to include other toolchain-specific parts.

```suggestion
    - cmake/*/arcmwdt/
    - include/zephyr/toolchain/mwdt.h
    - lib/libc/arcmwdt/*
```

---

Done.
Thanks for the suggestion.